app/presenters/publishing_api/base_item_presenter.rb

```diff
@@ -18,6 +18,7 @@ def base_attributes
         publishing_app: "whitehall",
         redirects: [],
         update_type: update_type,
+        auth_bypass_ids: [item.auth_bypass_id]
```

Comment:
```suggestion
        auth_bypass_ids: [item.auth_bypass_id],
```
You'll need a comma here to appease rubocop.